src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt

```diff
@@ -130,39 +108,24 @@ data class IndividualUser(
   private val _globalRoles = ResettableLazy { permissionStore.fetchGlobalRoles(userId) }
   override val globalRoles: Set<GlobalRole> by _globalRoles
 
-  /** History of permission checks performed in the current request or job. */
-  val permissionChecks: MutableList<PermissionCheck> = mutableListOf()
-
   override fun clearCachedPermissions() {
     _organizationRoles.reset()
     _facilityRoles.reset()
     _globalRoles.reset()
   }
 
   val fullName: String?
-    get() = makeFullName(firstName, lastName)
-
-  override fun <T> run(func: () -> T): T {
-    return CurrentUserHolder.runAs(this, func, authorities)
-  }
+    get() = TerrawareUser.makeFullName(firstName, lastName)
```

Comment:
Could move this to `TerrawareUser` too since I assume we'll want this for funder users.